Lib9c/TableData/StakeActionPointCoefficientSheet.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using static Nekoyume.TableData.TableExtensions;
+
+namespace Nekoyume.TableData
+{
+    [Serializable]
+    public class StakeActionPointCoefficientSheet : Sheet<int, StakeActionPointCoefficientSheet.Row>, IStakeRewardSheet
+    {
+        public IReadOnlyList<IStakeRewardRow> OrderedRows => OrderedList;
```

Comment:
Is `OrderedRows` necessary?
If you think the `OrderedList` is not intuitive, you can rename it.

Reply:
That is member of `IStakeRewardSheet`. Just a implementation 😉

Reply:
Oh, I didn't know about `IStakeRewardSheet`.😅